src/messages/ui.en.json

```diff
@@ -183,6 +183,7 @@
 
   "connectMB.pattern.heading": "Copy pattern",
   "connectMB.pattern.subtitle": "Copy the pattern displayed on the micro:bit.",
+  "connectMB.pattern.inputLabel": "Number of lit up LED in column {colNum} displayed on micro:bit",
```

Comment:
Will wait for Lucy to comment, but maybe this is more like "The height of illuminated LEDs in column {colNum} on the micro:bit"?